src/helpers/_typography.scss

```diff
@@ -98,15 +98,27 @@
   $font-map: map-get($govuk-typography-scale, $size);
 
   @each $breakpoint, $breakpoint-map in $font-map {
-    $font-size: map-get($breakpoint-map, "font-size") iff($important, !important);
-    $line-height: map-get($breakpoint-map, "line-height") iff($important, !important);
+    $font-size: map-get($breakpoint-map, "font-size");
+    $line-height: map-get($breakpoint-map, "line-height");
+
+    // Convert line-heights specified in pixels into a relative value, unless
+    // they are already unit-less (and thus already treated as relative values)
+    @if not unitless($line-height) {
```

Comment:
If you set your line-height to something other than 'px' would it this fail?

Reply:
Yep, because Sass doesn't know how to e.g. divide pixels by em.

We've got a few options…

- let Sass' error handling catch it
- detect if the units do not match and do nothing (output line-height as defined in the map)
- detect if the units do not match and throw our own error

Not sure what the best option is. Can you think of cases where you'd intentionally want to use define font-size and line-height in different units?

Reply:
I suppose the only case I'm thinking of is this could be confusing for people making their own font maps, but if it's clearly documented maybe it'll be fine.